ocs_ci/ocs/osd_operations.py

```diff
@@ -0,0 +1,229 @@
+import random
+import logging
+
+from semantic_version import Version
+
+from ocs_ci.framework import config
+from ocs_ci.utility.utils import get_ocp_version
+from ocs_ci.ocs.resources.ocs import get_job_obj
+from ocs_ci.ocs.resources.pvc import get_deviceset_pvs, get_deviceset_pvcs
+from ocs_ci.ocs import constants, node, ocp
+from ocs_ci.ocs import cluster
+from ocs_ci.ocs.resources.pod import (
+    get_osd_deployments,
+    get_osd_pods,
+    get_pod_node,
+    get_operator_pods,
+    get_osd_prepare_pods,
+    get_osd_pod_id,
+    run_osd_removal_job,
+    verify_osd_removal_job_completed_successfully,
+    delete_osd_removal_job,
+)
+from ocs_ci.helpers.sanity_helpers import Sanity
+
+
+logger = logging.getLogger(__name__)
+
+
+def osd_device_replacement(nodes):
+    """
+    Replacing randomly picked osd device
+    Args:
+        node (OCS): The OCS object representing the node
```

Comment:
Just one small comment... Need to change the name to `nodes`.

Reply:
ack